tests/e2e/ctl_v3_member_test.go

```diff
@@ -230,13 +236,29 @@ func ctlV3MemberUpdate(cx ctlCtx, memberID, peerURL string) error {
 	return e2e.SpawnWithExpectWithEnv(cmdArgs, cx.envMap, " updated in cluster ")
 }
 
-func TestCtlV3PromotingLearner(t *testing.T) {
+func TestCtlV3PromotingLearnerOnBootstrap(t *testing.T) {
 	e2e.BeforeTest(t)
 
+	testCtlV3PromotingLearner(t, 10)
+}
+
+func TestCtlV3PromotingLearnerOnApply(t *testing.T) {
+	e2e.BeforeTest(t)
+	// We call `LockOutsideApply` in apply, so we have to disable the verification.
+	revertFunc := verify.DisableVerification()
```

Comment:
Could we maybe just give an exception for stacktraces with `unsafeHackySaveMemberToBackend` ?

Reply:
good call & updated